### ISSUE TITLE
podman start does not work quite like docker start

### ISSUE THREAD

**Issue by rhatdan** [projectatomic/libpod#1017]
```
# docker create --name=stats_test_2g --memory=2g registry.access.redhat.com/rhel7 sleep 86400
191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
# docker start stats_test_2g
stats_test_2g
```
Versus
```
# podman create --name=stats_test1_2g --memory=2g registry.access.redhat.com/rhel7 sleep 86400
7810a4cfbfb0c3c5309391f6127f0886dff5a5b3d07f0bb187c97dc76f644996
# podman start stats_test1_2g
7810a4cfbfb0c3c5309391f6127f0886dff5a5b3d07f0bb187c97dc76f644996
```
Notice docker outputs the name of the container you told it to start whereas podman always tells you the UUID.

If I specify the UUID then docker will output the UUID, but if I specify the name docker outputs the name.  Podman should do the same. 
```
# docker start 191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
```

**Comment by rhatdan** [comment]
```
# docker create --name=stats_test_2g --memory=2g registry.access.redhat.com/rhel7 sleep 86400
191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
# docker start stats_test_2g
stats_test_2g
```
Versus
```
# podman create --name=stats_test1_2g --memory=2g registry.access.redhat.com/rhel7 sleep 86400
7810a4cfbfb0c3c5309391f6127f0886dff5a5b3d07f0bb187c97dc76f644996
# podman start stats_test1_2g
7810a4cfbfb0c3c5309391f6127f0886dff5a5b3d07f0bb187c97dc76f644996
```
Notice docker outputs the name of the container you told it to start whereas podman always tells you the UUID.

If I specify the UUID then docker will output the UUID, but if I specify the name docker outputs the name.  Podman should do the same. 
```
# docker start 191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
191c1b1086ebb23a63c60b5c757e3b3080c184286b404e5beca4d7294c47fa6c
```

**Comment by rhatdan** [comment]
@haircommander PTAL

**Comment by haircommander** [comment]
closed via #1019
